entity-framework/core/providers/sqlite/limitations.md

```diff
@@ -7,11 +7,11 @@ uid: core/providers/sqlite/limitations
 ---
 # SQLite EF Core Database Provider Limitations
 
-The SQLite provider has a number of migrations limitations. Most of these limitations are a result of limitations in the underlying SQLite database engine and are not specific to EF.
+The SQLite provider has a number of migrations limitations. Most of these limitations are a result of limitations in the underlying SQLite database engine and are not specific to EF Core.
```

Comment:
FYI we do routinely refer to EF Core as "EF" in our docs, for brevity. IIRC we sometimes say "EF Core" or even "Entity Framework Core" the first time in a page, and then just use "EF throughout".